projects/canopy/src/lib/modal/modal.module.ts

```diff
@@ -0,0 +1,51 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { A11yModule } from '@angular/cdk/a11y';
```

Comment:
Is there a way to only bring in the `cdkFocusTrap`? or is it all linked with the other `a11y` code, just concerned it might bloat the build size otherwise

Reply:
Good point, I can just include `CdkTrapFocus`

Reply:
Updated 👍

Reply:
I take this back, the build failed because the `cdkFocusTrap` is already imported by `A11yModule` so I will have to import all of it.